rules/general/error.js

```diff
@@ -4,7 +4,19 @@ module.exports = {
 
     // Ensure RegExp strings are valid
     // http://eslint.org/docs/rules/no-invalid-regexp
-    'no-invalid-regexp': 'error'
+    'no-invalid-regexp': 'error',
+
+    // Disallow undeclared variables
+    // https://eslint.org/docs/rules/no-undef
+    'no-undef': 'error',
+
+    // Disallow unused variables
+    // https://eslint.org/docs/rules/no-unused-vars
+    'no-unused-vars': 'error',
+
+    // Suggest using const
+    // https://eslint.org/docs/rules/prefer-const
+    'prefer-const': 'error'
```

Comment:
I'd suggest we put `prefer-const` under `es2015/style.js`.

Reply:
Ah, yes, that makes more sense, will move it 👍